Week2/homework/JS-Exercises/Exercise-4-The-reading-list.js

```diff
@@ -0,0 +1,22 @@
+let myBooks = [
+        book1 = {
+            title : "The Odyssey",
+            author : "Homer",
+            alreadyRead : true,
+        },
+        book2 = {
+            title : "Invisible Man",
+            author : "Ralph Ellison",
+            alreadyRead : false,
+        },
+        book3 = {
+            title : "The Stranger",
+            author : "Albert Camus",
+            alreadyRead : false,
+        }
+    ];
+
+    for ( let i = 0 ; i < myBooks.length ; i++) {
+        (myBooks[i].alreadyRead === true) ? console.log(myBooks[i].title + " by " + myBooks[i].author + " You already read this book.") : console.log(myBooks[i].title + " by " + myBooks[i].author + " You still need to read this book.");
+    };
+
```

Comment:
Couple things:
1. Could have used `const` instead of `let` while declaring `myBooks`.
2. This isn't the best use case of conditional operators on line 20. Ideally, we should use conditional operators when we need to do an assignment operation based on some condition. However, here we are doing a simple console.log statement which doesn't serve well the use case of conditional operators. A simple `if-else` block would make more sense here. 🙂 

A better use case of conditional operators: `const isItCold = (weather > 20) ? false : true;`